inlong-sort/sort-connectors/pom.xml

```diff
@@ -139,6 +144,12 @@
             <artifactId>hadoop-minicluster</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.streamnative.connectors</groupId>
+            <artifactId>pulsar-flink-connector_2.11</artifactId>
```

Comment:
The scala version of 2.11 can be replaced by variables?

Reply:
done